Basic/cordapp-example/build.gradle

```diff
@@ -25,6 +25,7 @@ buildscript {
         maven {
             url 'https://ci-artifactory.corda.r3cev.com/artifactory/corda-releases'
         }
+        maven { url 'https://ci-artifactory.corda.r3cev.com/artifactory/corda-dev/' }
```

Comment:
1. Peter to remove corda-dev repo links from all build.gradle/ or from repositories.gradle at the time of 4.6 release